tracker/tests.py

```diff
@@ -1,3 +1,65 @@
-from django.test import TestCase
-
+from django.test import TestCase, Client
+from django.urls import reverse, resolve
+import json
+from .models import *
+from .views import *
+import datetime
 # Create your tests here.
+
+class TestModels(TestCase):
+	"""docstring for ClassName"""
+	def setUp(self):
+		Task.objects.create(
+			name = 'xx',
+			planning = '07:04',
+			estimate = '02:50',
+			realtime = '04:52',
+			risk = 'OK',
+			level = 'easy',
+			notes = 'azerty',
+			estimateb_by_calc = '4:27',
+			correctness = 50,
+		)
+
+	def test_Task(self):
```

Comment:
Could you rename your test to be more explicit about what you are testing for?

Reply:
Since you have only one test in your class, you could just put the code inside setUp in your current test. If you plan to create an other test for example to check that the task is loaded in context data returned to home. It makes sense to have a setUp to do not duplicate code.